docs/sql-guide/statements/statement-insert-bulk.md

```diff
@@ -120,13 +120,36 @@ There are special assignments for certain literal values when inserting NDJSON d
 
 ## TRANSFORM clause
 
-* a list of valid SQL expressions that are used to specify data transformation before values are inserted.
-* uses variables named for the ordinal position values are specified in the MAP clause.
+The `TRANSFORM` clause is a list of valid SQL expressions that are used to specify data transformation before values are inserted. The variables to transform are named based on the ordinal position values are specified in the MAP clause.
 
 {: .important}
 The number of expressions in the column list and TRANSFORM clause must match.
 
-* [TRANSFORM example](/docs/sql-guide/statements/statement-insert-bulk/#transform-examples)
+### TUPLE() Function
```

Comment:
Is it possible to shift this content under the /docs/sql-guide/functions folder?

Reply:
That's basically a question I posed to @paddyjok above. As of now, it only works with bulk insert, so it doesn't make sense to put in functions

Reply:
tuple() is a scalar function like any other, the fact that it didn't work (would need to see an error) in a select should be a bug ticket.